screenshots.html

```diff
@@ -0,0 +1,114 @@
+<html>
+    <head>
+        <title> images</title>
+      <style>
+                   
+            .content
+                {
+                    margin-inline-start: 1%;
+                    float: left;
+                    height: 350px;
+                    position: relative;
+                    padding: 0px 20px 0px 30px;
+                    margin-left: -75px;
+                    margin-top:-10px;
+                    top: 0;
+                    margin-right:60px;
+                    border-right: 1px solid rgb(208, 212, 216);
+                }
+            .last_content
+                {
+                    margin-inline-start: 1%;
+                    float: left;
+                    position: relative;
+                    padding: 0px 20px 0px 30px;
+                    margin-left: -75px;
+                    margin-top:-10px;
+                    top: 0;
+                    margin-right:60px;
```

Comment:
last_content and content have almost the same style definition exception for border-right.   Violates DRY princple -https://code.tutsplus.com/tutorials/3-key-software-principles-you-must-understand--net-25161

You can always use the last-child pseudo class to remove the border.

Reply:
https://google.github.io/styleguide/htmlcssguide.html .  Follow this guidelines when writing html/css